src/goose/defaults.clj

```diff
@@ -36,6 +36,24 @@
 (def ^:no-doc prefixed-cron-queue (prefix-queue cron-queue))
 (def ^:no-doc prefixed-cron-entries (str "goose/" cron-entries))
 
+;;; ======== Batch Namespace =======
+
+(def ^:no-doc batch-prefix "goose/batch:")
+
+(def enqueued-job-set "enqueued")
+(def successful-job-set "successful")
+(def retrying-job-set "retrying")
+(def dead-job-set "dead")
+
+(defn ^:no-doc prefix-batch
+  [batch-id]
+  (str batch-prefix batch-id))
+
+(defn ^:no-doc batch-set
```

Comment:
```suggestion
(defn ^:no-doc prefix-batch-set
```

Reply:
Realised that all the function constructing key name consist of prefix. I will add the change.